etc/env/example.env

```diff
@@ -226,31 +219,36 @@ KEYCLOAK_REALM=candig
 KEYCLOAK_CLIENT_ID=local_candig
 KEYCLOAK_LOGIN_REDIRECT_PATH=/auth/login
 KEYCLOAK_PORT=8080
-#TODO: change remaining KEYCLOAK_SERVICE_* to KEYCLOAK_*
-KEYCLOAK_SERVICE_CONTAINER_PORT=8080
-KEYCLOAK_SERVICE_HOST=0.0.0.0
-KEYCLOAK_SERVICE_PUBLIC_PROTO=http
-KEYCLOAK_SERVICE_PRIVATE_PROTO=http
-KEYCLOAK_SERVICE_PUBLIC_URL=${KEYCLOAK_SERVICE_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_PORT}
-KEYCLOAK_SERVICE_PRIVATE_URL=${KEYCLOAK_SERVICE_PRIVATE_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_SERVICE_CONTAINER_PORT}
+KEYCLOAK_CONTAINER_PORT=8080
+KEYCLOAK_HOST=0.0.0.0
+KEYCLOAK_PUBLIC_PROTO=http
+KEYCLOAK_PRIVATE_PROTO=http
+KEYCLOAK_PUBLIC_URL=${KEYCLOAK_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_PORT}
+KEYCLOAK_PUBLIC_URL_PROD=${KEYCLOAK_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}
+KEYCLOAK_PRIVATE_URL=${KEYCLOAK_PRIVATE_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_CONTAINER_PORT}
 
+KEYCLOAK_GENERATE_TEST_USER=0
 
 # tyk service
-TYK_VERSION=v2.9.3.1
-TYK_REDIS_VERSION=4.0.14-alpine
+TYK_VERSION=v3.2
+TYK_REDIS_VERSION=5.0-alpine
 TYK_SERVICE_PUBLIC_PORT=5080
 TYK_SERVICE_HOST=0.0.0.0
 #TODO: consolidate tyk public and private domains
-TYK_TARGET_URL=http://${CANDIG_ARBITER_HOST}:${CANDIG_ARBITER_SERVICE_PORT}
-TYK_LOGIN_TARGET_URL=http://${TYK_SERVICE_HOST}:${TYK_SERVICE_PUBLIC_PORT}
+TYK_LOGIN_TARGET_URL=http://${CANDIG_DOMAIN}:${TYK_SERVICE_PUBLIC_PORT}
+TYK_ANALYTICS_FROM_EMAIL=admin@distributedgenomics.ca
+TYK_ANALYTICS_FROM_NAME=CanDIG Admin
 TYK_LISTEN_PATH=
 TYK_POLICY_ID=candig_policy
+## api - authentication
 TYK_AUTH_API_ID=11
 TYK_AUTH_API_NAME=Authentication
 TYK_AUTH_API_SLUG=authentication
+## api - candig-server (v1)
 TYK_CANDIG_API_ID=21
 TYK_CANDIG_API_NAME=CanDIG
 TYK_CANDIG_API_SLUG=candig
+TYK_CANDIG_API_TARGET=http://${CANDIG_DOMAIN}:${CANDIG_SERVER_PORT}
```

Comment:
you are still using candig-server env vars

Reply:
yes they are define here https://github.com/CanDIG/CanDIGv2/blob/amanjeev/dig-633-authentication/etc/env/example.env#L193-L198